backend/handlers/handlers.go

```diff
@@ -702,11 +702,16 @@ func (handlers *Handlers) postBtcFormatUnit(r *http.Request) interface{} {
 }
 
 // getAccountsBalanceHandler returns the balance of all the accounts, grouped by keystore and coin.
-func (handlers *Handlers) getAccountsBalance(*http.Request) (interface{}, error) {
+func (handlers *Handlers) getAccountsBalance(*http.Request) interface{} {
+	type response struct {
+		Success     bool                                                     `json:"success"`
+		ErrorCode   string                                                   `json:"errorCode,omitempty"`
+		TotalAmount map[string]map[coin.Code]accountHandlers.FormattedAmount `json:"formattedAmount,omitempty"`
+	}
 	totalAmount := make(map[string]map[coin.Code]accountHandlers.FormattedAmount)
 	accountsByKeystore, err := handlers.backend.AccountsByKeystore()
 	if err != nil {
-		return nil, err
+		return response{Success: false, ErrorCode: err.Error()}
 	}
```

Comment:
You are again putting an error that might not be an error code into ErrorCode, here an in all error responses below.

Reply:
Removed the errocode in the api as discussed